composer.json

```diff
@@ -12,17 +12,17 @@
     "extra": {
         "branch-alias": {
             "dev-master": "1.0-dev",
-            "dev-develop": "1.1-dev"
+            "dev-develop": "2.0-dev"
         }
     },
     "require": {
         "php": "^5.6 || ^7.0",
         "container-interop/container-interop": "^1.1",
         "psr/http-message": "^1.0",
         "zendframework/zend-diactoros": "^1.1",
-        "zendframework/zend-expressive-router": "^2.0",
+        "zendframework/zend-expressive-router": "^2.1",
```

Comment:
This was necessary, because before 2.1, we only allowed callables for the `$middleware` argument to the `Route` constructor. 2.1 allows http-interop/http-middleware 0.4.1 values as well.